changelog/index.html

```diff
@@ -77,7 +77,29 @@ <h5> Paleomagnetism.org 2 - Changelog </h5>
         </div>
 
         <div id="accordion">
+   <div class="card">
+            <div class="card-header" id="headingNine">
+              <button class="btn btn-link" data-toggle="collapse" data-target="#collapseNine" aria-expanded="true" aria-controls="collapseNine">
```

Comment:
collapseNine/headingNine should become collapseTen everywhere. Otherwise it references the other card below. Kinda annoying but eh

Reply:
That makes complete sense. But I am a complete idiot.